docs/window.dox

```diff
@@ -892,6 +893,9 @@ the user or with @ref glfwIconifyWindow.
 `GLFW_MAXIMIZED` indicates whether the specified window is maximized, whether by
 the user or with @ref glfwMaximizeWindow.
 
+`GLFW_CENTER_CURSOR` indicates whether the cursor should be moved to the center
+of the window during creation.
```

Comment:
`of the full screen window during creation.  This is ignored for windowed mode windows.` or something like that.